### DISCUSSION TITLE
Korean can't in google

### DISCUSSION THREAD

**Discussion by monteeys** [DandelionSprout/adfilt#222] [Q&A]
Whenever type Korean(Hangul) into the Google search box, the characters are broken so cannot enter Korean.

**Comment by ghost** [comment]
There is a similar report on the Actually Legitimate URL Shortener Tool but I couldn't reproduce it.
`https://egg.5ch.net/test/read.cgi/android/1628427164/851`

**Reply by DandelionSprout** [reply]
Seeing as the OP (monteeys) provided non-existent info on the lists (s)he used, or which OS/device they used, I had no real way of doing anything with this report at the time.
